docs/cody/capabilities/ignore-context.mdx

```diff
@@ -1,10 +1,151 @@
 # Cody Ignore
 
-<Callout type="note">This feature is currently in the Experimental stage and only supported with VS Code extension.</Callout>
+## Enterprise
 
-This guide will walk you through the process of ignoring specific files or folders in your codebase when using Cody. 
+<Callout type="note">
+Client Support for Cody Context Filters
 
-## Enabling Unstable Features
+-   Cody context filters are currently supported only by Cody Web.
+-   Starting from May 27, support for Cody context filters will also be available on the Cody VS Code Extension and the Cody JetBrains Extension.
+-   A minimum required client version is enforced for these extensions: `>=1.20.0` for the VS Code extension and `>=6.0.0` for the JetBrains extension.
+-   If Cody context filters are defined in the site configuration, other Cody clients will not function as they do not support this feature.
+-   The feature is disabled by default until the required extensions versions are released. Administrators wishing to enable Cody context filters must activate the "cody-context-filters-enabled" feature flag. Be aware that enabling this feature may prevent VS Code and JetBrains clients that do not meet the version constraint from working.
+
+</Callout>
+
+Enterprise instances admins can define specific rules to determine which repositories Cody is allowed to utilize for context in its requests to third-party LLMs.
+
+To establish these rules, administrators should configure the `cody.contextFilters` field in the site configuration.
+
+The field structure is as follows:
+
+```json
+{
+	"cody.contextFilters": {
+		"include": [
+			{
+				"repoNamePattern": "^github\\.com\\/sourcegraph\\/.*"
+			}
+		],
+		"exclude": [
+			{
+				"repoNamePattern": ".*analytics.*"
+			}
+		]
+	}
+}
```

Comment:
@taras-yemets I recommend throwing in comments to describe what each rule here does :)